### PR TITLE
Added configuration for Notification demo Android

### PR DESCRIPTION
**Pull request by Slash32**

### CODE REVIEW

**Comment by Slash32** [comment]
@kaa-leeroy test this fully please

File: notificationdemo/resources/projects.xml [hunk 1]
```diff
@@ -259,6 +259,40 @@ Try the following:
...
+        <!-- Specifies the target runtime platforms according to the instructions from the project details section
+             (ex. LINUX_X86, ANDROID, IOS, ESP8266, etc., to get list of all possible options
+             see 'platform' type in demo_projects_schema.xsd schema) -->
+
+        <platforms>ANDROID</platforms>
+        <features>NOTIFICATION</features>
+        <complexity>BASIC</complexity>
+        <bundleId>notification_demos</bundleId>
+        <sourceArchive>android/notification_demo.tar.gz</sourceArchive>
+        <projectFolder>AndroidNotificationDemo</projectFolder>
```
Comment (snitsaruk) [inline notificationdemo/resources/projects.xml]: `<projectFolder>notification_demo/NotificationDemo</projectFolder>`

File: notificationdemo/resources/projects.xml [hunk 2]
```diff
@@ -259,6 +259,40 @@ Try the following:
...
+        <sdkLibDir>AndroidNotificationDemo/libs</sdkLibDir>
```
Comment (snitsaruk) [inline notificationdemo/resources/projects.xml]: `<sdkLibDir>notification_demo/NotificationDemo/app/libs</sdkLibDir>`

File: notificationdemo/resources/projects.xml [hunk 3]
```diff
@@ -259,6 +259,40 @@ Try the following:
...
+        <destBinaryFile>AndroidNotificationDemo/bin/AndroidNotificationDemo-debug.apk</destBinaryFile>
```
Comment (snitsaruk) [inline notificationdemo/resources/projects.xml]: `<destBinaryFile>notification_demo/NotificationDemo/app/build/outputs/apk/app-debug.apk</destBinaryFile>`

**Review by snitsaruk — COMMENTED** [review]
Added config in comments for Notification Demo android app

**Comment by Slash32** [comment]
The commits chain has already passed Jenkins as https://github.com/kaaproject/sample-apps/pull/322 before merge to this branch.

**Pull request merged**